docs/doc/11-integrations/00-api/02-clickhouse-handler.md

```diff
@@ -2,14 +2,14 @@
 title: ClickHouse Handler
 sidebar_label: ClickHouse Handler
 description:
-  Databend is ClickHouse wire protocol-compatible.
+  Databend is ClickHouse HTTP API wire protocol-compatible.
 ---
 
 ![image](/img/api/api-handler-clickhouse.png)
 
 ## Overview
 
-Databend is ClickHouse wire protocol-compatible, allow you to connect to Databend server with Clickhouse client, make it easier for users/developers to use Databend.
+Databend is ClickHouse HTTP API wire protocol-compatible, allowing you to connect to Databend with ClickHouse HTTP Handler, make it easier for users/developers to use Databend.
```

Comment:
I think we can get rid of the part "make it easier for users/developers to use Databend.". It makes the whole sentence grammatically incorrect. 
If you need it anyway, we can say:

..., allowing users and developers to easily connect to Databend with ClickHouse HTTP Handler.